vcd/resource_vcd_security_tag.go

```diff
@@ -0,0 +1,113 @@
+package vcd
+
+import (
+	"context"
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/vmware/go-vcloud-director/v2/types/v56"
+)
+
+func resourceVcdOpenApiSecurityTag() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceVcdOpenApiSecurityTagCreate,
+		ReadContext:   resourceVcdOpenApiSecurityTagRead,
+		UpdateContext: resourceVcdOpenApiSecurityTagCreate,
+		DeleteContext: resourceVcdOpenApiSecurityTagDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: resourceVcdOpenApiSecurityTagImport,
+		},
+
+		Schema: map[string]*schema.Schema{
```

Comment:
I think we're missing `org` field as I can create a security tag in one org, but don't see it in other - that means that it is somehow bound to a particular Org I created it in.